packages/host/webpack.config.js

```diff
@@ -16,6 +16,7 @@ module.exports = env => {
   return {
     entry: './src/index',
     mode: isProduction ? 'production' : 'development',
+    devtool: 'source-map',
```

Comment:
Can revert this - it's making the initial build a little longer but gives better source maps for debugging in chrome. Might want to change it to hidden source maps for production builds